Main/Source/iconf.cpp

```diff
@@ -59,108 +59,108 @@ scrollbaroption ivanconfig::Contrast(     "Contrast",
                                           &ContrastChanger,
                                           &ContrastHandler);
 cycleoption ivanconfig::HitIndicator(     "HitIndicator",
-                                          "Show Hit",
+                                          "show successful hit",
                                           0, 5,
                                           &HitIndicatorDisplayer);
 cycleoption ivanconfig::HoldPosMaxDist(   "HoldPosMaxDist",
-                                          "Pet will wait near the last position", // if pet is set to not follow, will move away max the specified distance. if it loses the player, will stay near the last position it moves to trying to follow the player.
+                                          "pets stay put when not following", // if pet is set to not follow, will move away max the specified distance. if it loses the player, will stay near the last position it moves to trying to follow the player.
```

Comment:
btw, that extra help info (commented) could be shown (despite not really needed in this specific case) after aplying this patch 
https://github.com/AquariusPower/ivan/pull/47/commits/bfffc75707f00191c64194970b9bb81f08d36b92#diff-0955c50bbcbd4273d41fd3751e41d05fR186